src/EditorFeatures/CSharpTest/Completion/CompletionProviders/TypeImportCompletionProviderTests.cs

```diff
@@ -1835,6 +1796,49 @@ public async Task TestGlobalUsingsInUserDocument()
             await VerifyTypeImportItemIsAbsentAsync(markup, "Console", inlineDescription: "System");
         }
 
+        [Theory]
+        [InlineData(null)]
+        [InlineData(true)]
+        [InlineData(false)]
```

Comment:
Combinatorial doesn't work for nullable?  that's a pity.

Reply:
yup, took me a few minutes to realize that's the cause of test failure like this

```
00.93]       System.NotSupportedException : Specified method is not supported.
[xUnit.net 00:00:00.93]       Stack Trace:
[xUnit.net 00:00:00.93]            at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
[xUnit.net 00:00:00.94]            at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
```